### DISCUSSION TITLE
typescript-apollo-angular's apollo federation support

### DISCUSSION THREAD

**Discussion by charlesxsh** [dotansimha/graphql-code-generator#5708] [Q&A] [answered]
Is plugin 'typescript-apollo-angular' going to support federation? Seems like typescript-apollo-angular will fail if schemas are GraphQL files with federation.

**Comment by dotansimha** [comment, marked as answer]
Federation is used for defining your schema in a distributed way. Eventually, in runtime, it's being merged and running together as a regular GraphQL schema. 
From the point of view of Angular (or, any frontend) developer, it doesn't matter if the schema is built with Federation or not. You can just consume it as a regular schema.
